### PR TITLE
NFDIV-714: Add CaseEventFieldLabel and CaseEventHintText as required

### PR DESCRIPTION
**Pull request by tomxelliott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-714

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ plugins {
   id 'io.spring.dependency-management' version '1.0.11.RELEASE'
   id 'org.springframework.boot' version '2.4.5'
   id 'com.github.ben-manes.versions' version '0.38.0'
-  id 'hmcts.ccd.sdk' version '0.20.3'
+  id 'hmcts.ccd.sdk' version '0.20.4'
```
Comment (nitinprabhu) [inline build.gradle:21]: there is 0.20.5 now :)

**Review by nitinprabhu — COMMENTED** [review]
Are the changes tested locally ?

File: build.gradle (hunk 1 above)
Comment (tomxelliott) [reply build.gradle:21]: Will upgrade

File: src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java [hunk 2]
```diff
@@ -240,14 +240,27 @@
             + "marriage certificate?",
         access = {DefaultAccess.class}
     )
-    private YesOrNo applicant2NameAsOnMarriageCertificate;
+    private YesOrNo applicant2NameDifferentToMarriageCertificate;
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java:243]: does this require a case type ID bump and frontend patch?

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**